Updated some wording across readme (#5)
worldbank/dime-github-trainings: GitHub-resources/DIME-GitHub-Guides/Creating-GitHub-account.md

  1. Click emails in the menu (or click here https://github.com/settings/emails)
  1. In the field that says _Add email address_ enter the email you want to add as your second email. An email will be sent to that address with instructions for how to confirm that you own that email. Complete those instructions before doing next step.
* **Set up notification for the second email**

>   1. After you have added a second email, click _Notifications_ in the settings menu (or click here https://github.com/settings/notifications). Make sure you are still logged into github.com from your personal email.

Author: @roshni13khincha , your account has two emails associated to it now, but you can still only be logged in or not on account level, not on email level. See my suggestion below, does that better clarify what I mean?
```suggestion
  1. After you have added a second email, click _Notifications_ in the settings menu (or click here https://github.com/settings/notifications, link only works if you are logged in to GitHub.com).
```

Author: Click `Commit suggestion` to accept the suggestion as a commit directly to the PR
